app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ActiveRecord::Base
+  has_many :lists
+
+  validates :title, presence: true
```

Comment:
The only validation that I think would be safe to assume is validating that the start date is after the end date.